### ISSUE TITLE
[BUG?] Selecting column with alias makes delete row impossible

### ISSUE THREAD

**Issue by wiwwo** [dbeaver/dbeaver#8131]
Not sure it is a bug...
Imagine doing something like
`select orderid, cusomerid from orders`
Then, on datagrid table, delete one row. It works (well, might ask "what is table key etc, but... it works").

Now, on same table, do 
`select orderid as ord, cusomerid from orders`
You cannot delete any row anymore:
```
Error synchronizing data with database

Reason:
SQL Error [42703]: ERROR: column "ord" does not exist
  Hint: Perhaps you meant to reference the column "orders.orderid".

```

**Comment by uslss** [comment]
thanks for the bug report

**Comment by serge-rider** [comment]
Can't reproduce. 
@wiwwo what is your database?

**Comment by wiwwo** [comment]
Postgresql.
It is an editor problem i think, not a DB one... Maybe internally the grid calls `update TABLE set ord = ...`, referring to the alias, instead of `update TABLE set orderid = ...`

**Comment by wiwwo** [comment]
Seems to be fixed in 7.0(.1). Now column is readonly...
